src/App.js

```diff
@@ -1,36 +1,29 @@
-import react from 'react';
+import { useState } from 'react';
 import Calculator from './components/Calculator';
 import calculate from './logic/calculate';
 
-class App extends react.Component {
-  constructor(props) {
-    super(props);
-    this.state = { total: null, next: null, operation: null };
-    this.updateDetails = this.updateDetails.bind(this);
-  }
+function App() {
```

Comment:
- [ ] <b>REQUIRED:<b/> Prefer using ES6 arrow function. Actually, there is nothing wrong with ES5 functions syntax but it is mandatory to use es6 features in the javascript module. besides, Arrow functions provide a simplified structure, allowing for a more concise way to write function expressions. which makes them better than ES5 functions(function declarations). Here is the syntax of an ES6 arrow function...
```javascript
  const functionName = () => {
     // code here
  }
```